misc/profiles2/lookups.dat

```diff
@@ -69,123 +59,245 @@ surface;0000005778 mud
+estimated_noise_class;0000000001 2
+estimated_noise_class;0000000001 3
+estimated_noise_class;0000000001 4
+estimated_noise_class;0000000001 5
+estimated_noise_class;0000000001 6
+
+estimated_river_class;0000000001 1
+estimated_river_class;0000000001 2
+estimated_river_class;0000000001 3
+estimated_river_class;0000000001 4
+estimated_river_class;0000000001 5
+estimated_river_class;0000000001 6
+
+estimated_town_class;0000000001 1
+estimated_town_class;0000000001 2
+estimated_town_class;0000000001 3
+estimated_town_class;0000000001 4
+estimated_town_class;0000000001 5
+estimated_town_class;0000000001 6
 
-maxspeed;0001058313 50 30_mph 30mph
-maxspeed;0000860780 30 20_mph 20mph
-maxspeed;0000025232 10 5 7 15
-maxspeed;0000083989 20 10_mph 10mph 15_mph 15mph
+maxspeed;0001058313 50 55 56 30_mph 30mph
+maxspeed;0000860780 30 35 20_mph 20mph
+maxspeed;0000025232 10 15 5 6 8 5_mph 6_mph
+maxspeed;0000083989 20 25 10_mph 10mph 15_mph 15mph
 maxspeed;0000195097 40 45 25_mph 25mph
-maxspeed;0000204646 60 35_mph 35mph 40_mph 40mph
-maxspeed;0000130108 70 45_mph 45mph
-maxspeed;0000225071 80 50_mph 50mph
-maxspeed;0000106719 90 55_mph 55mph
+maxspeed;0000204646 60 65 35_mph 35mph 40_mph 40mph
+maxspeed;0000130108 70 75 45_mph 45mph
+maxspeed;0000225071 80 85 50_mph 50mph
+maxspeed;0000106719 90 95 55_mph 55mph
 maxspeed;0000134522 100 60_mph 60mph 65_mph 65mph
 maxspeed;0000025242 110 70_mph 70mph
 maxspeed;0000038763 120 75_mph 75mph
-maxspeed;0000026953 130
+maxspeed;0000026953 130 80_mph 79_mph
 maxspeed;0000138654 urban RO:urban RU:urban FR:urban IT:urban AT:urban DE:urban UA:urban
 maxspeed;0000138654 rural RO:rural RU:rural FR:rural IT:rural AT:rural DE:rural UA:rural
 
 service;0001433919 parking_aisle
-service;0001305879 driveway
+service;0001305879 driveway driveway2
 service;0000382788 alley
 service;0000018777 drive-through drive_through
 service;0000008290 emergency_access
 service;0000003138 bus
 service;0000001250 parking
 service;0000001159 logging
+service;0000000001 yard
+service;0000000001 spur
+service;0000000001 siding
+service;0000000001 crossover
+
+tracktype;0000887965 grade2
+tracktype;0000868414 grade3
+tracktype;0000595882 grade1
+tracktype;0000568372 grade4 grade3-5
+tracktype;0000405959 grade5
+
+access;0002688349 private restricted residents employees
+access;0000319927 yes public
+access;0000144799 no
+access;0000140215 permissive
+access;0000108802 destination visitors
+access;0000099899 agricultural forestry agricultural;forestry
+access;0000039934 designated official
+access;0000011813 customers
+access;0000004007 delivery
+access;0000000100 psv
+access;0000000100 hov
+access;0000000001 permit
+access;0000000001 military
+
+estimated_traffic_class;0000000001 1
+estimated_traffic_class;0000000001 2
+estimated_traffic_class;0000000001 3
+estimated_traffic_class;0000000001 4
+estimated_traffic_class;0000000001 5
+estimated_traffic_class;0000000001 6
+estimated_traffic_class;0000000001 7
 
 lit;0000809223 yes
```

Comment:
I'd like to add `lit=no` which would allow to avoid ways that are known to unlit.

Background: I'm developing a pedestrian profile currently and would like to provide an option "Avoid unlit ways".

Reply:
Great that `highway;0000000001 elevator` in the way context is finally here. That will improve routing through the Kennedyfietstunnel in Antwerp.